lips/lip-6.mdx

```diff
@@ -0,0 +1,140 @@
+* *Validator Set* is an on-chain resource that stores a vector, where each entry contains an address of a Validator Owner and a copy of its Validator Config. Though the Validator Config in the Validator Set may be an older version of the Validator Config currently stored under Validator Owner's account, each Validator Node is run with the Validator Config from the Validator Set.
+Each address appears at most once in the vector. The Libra Association collectively decides which Validator Owners' addresses are in the Validator Set. There is a one-to-one correspondence between a Validator Owner in the Validator Set and a Validator Node. The Validator Set changes, either when one of the Validator Nodes updates their parameters or when a new member is added or removed, the Reconfiguration happens.
+* *Validator Node* is a replica that runs the Libra Protocol to maintain the blockchain state. A Validator Node processes transactions and interacts with other Validator Nodes to reach consensus on the ordering of transactions and the resulting state of the database after transactions' executions.
+* *Validator Operator* is an owner of an on-chain Libra account, it may operate none, one or several Validator Nodes: a Validator Owner may designate a Validator Operator to run a Validator Node on Validator Owner's behalf. A Validator Operator may run multiple Validator Nodes. The Validator Operator for a Validator Owner has permission to rotate the Validator Config for the Validator Owner locally, as well as copy it to the Validator Set.
+* *Reconfiguration* defines a new epoch by triggerring a new epoch event (`LibraConfig::NewEpochEvent`), it gets emitted if any part of the global on-chain configuration, such as the Validator Set, changes. If a transaction causes an epoch change, that transaction is the last transaction committed in the current epoch — any subsequent transactions in that block or future blocks from that epoch will be ignored. Once the transaction has been committed, the new configuration starts being valid immediately. E.g. the new Validator Set will start running the consensus protocol in the new epoch.
+
+<img alt="Validator Set" src={useBaseUrl('img/validator_set.svg')} />
+
+# Specifications
+
+## Workflows
+### Validator Owner and Validator Operator accounts creation
+Only the Association (through the Association Operation Services) may create Validator Owner and Validator Operator accounts.
+Validator Owners and Validator Operators should contact the Association off-chain and communicate their authentication keys as well as the names of their organizations written in snake case (lowercase, where spaces are replaced with underscores). The authentication key is a 32 elements byte array, derived from the public verification key. The protocol currently supports Ed25519 and MultiEd25519 signature schemes.
```

Comment:
not in this revision, but we need to be clear that some accounts are self-managed and eventually all will be but some are also managed by the association on behalf of owners. these accounts have an auth key of all 0 and never directly submit transactions to the chain but instead are controlled via libra root ... or something like that.

Reply:
There is a special Note section at the end of the doc on that.

Reply:
seems to be addressed in the Notes section at the bottom?

Reply:
that is technically true but I prefer to see it unified so it doesn't feel so ... much as an after thought as this special note applies to /every/ validator owner in genesis.

Reply:
I moved the sentences about libra_root managing the validator owners up into the Workflows section.